dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/MetadataReportConfig.java

```diff
@@ -16,13 +16,13 @@
  */
 package org.apache.dubbo.config;
 
-import org.apache.dubbo.common.Constants;
+import org.apache.dubbo.common.constants.CommonConstants;
 import org.apache.dubbo.common.utils.StringUtils;
 import org.apache.dubbo.config.support.Parameter;
 
 import java.util.Map;
 
-import static org.apache.dubbo.common.Constants.PROPERTIES_CHAR_SEPERATOR;
+import static org.apache.dubbo.common.constants.CommonConstants.PROPERTIES_CHAR_SEPERATOR;
```

Comment:
org.apache.dubbo.common.constants.CommonConstants has been imported here,
choose to import static  org.apache.dubbo.common.constants.CommonConstants.DUBBO 
or modify PROPERTIES_CHAR_SEPERATOR to CommonConstants.PROPERTIES_CHAR_SEPERATOR